packages/babel-preset-mc-app/index.js

```diff
@@ -102,9 +103,10 @@ module.exports = function getBabePresetConfigForMcApp() {
       isEnvProduction && [
         // Remove PropTypes from production build
         require('babel-plugin-transform-react-remove-prop-types').default,
-        {
-          removeImport: true,
-        },
+        // In case of rollup bundles, we want to keep the prop types but wrap
+        // them into a `process.env.NODE_ENV !== "production"` so that when
+        // building the final application bundles, those codes parts can be removed.
+        isRollup ? { mode: 'wrap' } : { removeImport: true },
```

Comment:
Given that prop-types were never be made available for consumers this is not breaking. Just for completeness, otherwise it would as so,e may have built components by reading prop-types.